mkdocs.yml

```diff
@@ -100,3 +102,8 @@ markdown_extensions:
       options:
         custom_icons:
           - site/overrides/.icons
+
+plugins:
+    - redirects:
+        redirect_maps:
+            'docs/installing.md': 'docs/getting-started/install.md'
```

Comment:
do we need to redirect `docs/getting-started.md: 'docs/getting-started/first-app-tutorial.md'` as well?